cmd/input-file-available-producer/main.go

```diff
@@ -0,0 +1,41 @@
+package main
```

Comment:
compare and contrast these commands with this one:
https://github.com/ONSdigital/dp-import/pull/1/files#diff-e5a298153f63173f49aac58a07081eacR1
think I prefer yours 😂